### ISSUE TITLE
Changelogs

### ISSUE THREAD

**Issue by bGZo** [bGZo/blog#1]
### No emotion; no journal; less platitude

I have a habit of writing diaries in high school, to decrease the pressure and anxiety. So blog inherit it. As time going by, I realize I have to make a review of my life, in every single post, and it's really, kind of wasting time. Digging feeling or writing complain make every post like a toilet paper. Emotion and cliché may means a lot to the present, but less in the future[^3].

### No necessary to rewrite blog

You have to read old articles to find the mistakes one by one, even word alternatives, sentences order and font size or spaces. It's such a hell. So why not delete above all, then write a new one? Interestingly, this scenes sounds like *Bojack Horseman* wrote his first book[^7]:

![chrome_vkyHRlDwRf](https://user-images.githubusercontent.com/57313137/211231511-518066e9-7184-4348-93fb-bdd2b4862b54.png)

### Show comments wontfix

1. It's really due to lazy, maybe a while, more likely wouldn't. 
2. I want this design being a filter to tired words in readers. Sounds like GFW we known, actually it's true. This design would be a soft-wall said by haohailong[^8]
3. People die, but long live github.[^4] However github is the third free services as well, like [disqus](https://disqus.com), [vercel](https://vercel.com) or something else. I don't trust them, free forever? Don't lie, nothing is free, sometimes it paid more than given. Backup as schedule is more reliability, and find a more sensible platform. Alternative is, creating it.
4. The function-added would be left to github, like action, comments, and more difficult problem in the future (;

## Logs Reversed 

- 20250212 
  - [Feat] Delete all thoughts and letters, move them to [quartz](https://github.com/bGZo/notes/) and Obsidian.
  - [Feat] Microblog powered by TG added.
- 20230109
  - [Feat] Update [mathjax - npm](https://www.npmjs.com/package/mathjax ) library from version 2 to 3.
  - [Docs] Archived old article and add in this one.
- 20230101
  - [Feat] Plugin [jekyll-seo-tag](https://jekyll.github.io/jekyll-seo-tag/) supported.
- 20221222
  - [Style] Modify the style of theme to [solarized](https://ethanschoonover.com/solarized), mostly color of theme;
  - [Style] Add the table of contents and footnotes support;
  - [Style] Change emoji icon to normal icon for compatibility, using [Favicon Generator](https://realfavicongenerator.net/)[^6];
  - [Feat] Add [UNPKG](https://unpkg.com) CDN Support;
  - [Feat] Add back to the top named `ikaros`[^5];
  - [Docs] Rewrite changelogs;
- 20220521
  - Hosted by github issues with github action;
- 20220121
  - Latex supported by [MathJax](https://www.mathjax.org);
  - Host by [Vercel](https://vercel.com);
  - Details via: [commit](https://github.com/bGZo/blog/commit/d2430f81effebb4d8a15db205203564bd865b08027);
- 20220104
  - The fifth blog using [Jekyll](https://jekyllrb.com) build-in github;
  - [Style] Switch to theme imitating [@dzhavat](https://github.com/dzhavat/dzhavat.github.io);
  - Details via: [commit](https://github.com/bGZo/blog/commit/20428035c7167ce2899e4db9fb5d1d006d60829cc3);
- 20210607
  - [Broken] Delete the whole blog by accident in WSL, but you still could subscribe old article via rss link I [archived](https://unpkg.com/bgzo/file/4blog.xml);
  - [Style] Switch to [Hugo-tailwind-journal](https://github.com/ianrodrigues/hugo-tailwind-journal);
    ![chrome_7T2ErRaYW6-or8](https://user-images.githubusercontent.com/57313137/211230541-2515d7d8-770d-45d1-8840-f91cdbc37352.png)
- 20210506
  - [Style] Site icon begin to use emoji[^1];
- 20210114
  - [Broken] Give up Censored MIIT (as ICP) absolutely;
- 20210112
  - The forth blog using [Hugo](https://gohugo.io) & [theme-MemE](https://github.com/reuixiy/hugo-theme-meme);
    ![chrome_vf7lFehtYE-or8-or8](https://user-images.githubusercontent.com/57313137/211234442-0621d2cf-0b0a-45e9-801a-08d043df5962.png)
- 20200824
  - [Broken] Delete old github.io repo, there is no backup before. So the before blog would missing forever 😭
- 20200901
  - [Broken] Switch directories and find notes suck, so give up note using github repo;
    - Appended at 221223: Actually using [VSCode](https://code.visualstudio.com) search could solve this problem. It's full-text search 🤩
- 20200613
  - Switch theme to [hexo-theme-Wikitten](https://github.com/zthxxx/hexo-theme-Wikitten);
- 20200605
  - The third blog using [Typecho](https://typecho.org/);
  - Images hosted by [Chevereto](https://chevereto.com/)
  - [Style] Modify theme using PHP;
- 20200420
  - Domain change so register at MIIT (as ICP) again;
- 20200402
  - [Style] Switch theme to [aircloud](https://github.com/aircloud/hexo-theme-aircloud);
- 20200214
  - [Broken] Censored sucks. Give up [CODING.NET](https://coding.net);
  ![codingnet](https://user-images.githubusercontent.com/57313137/209327464-13740ca4-0b8c-4dbe-9d9c-2fbe4b2ff27d.png)
- 20200113
  - The second blog using [WordPress](https://wordpress.com);
- 20200112
  - Register at MIIT (as ICP) in China;
- 20191223
  - The first blog using [Hexo](https://hexo.io);

### Back tp top

<p class="codepen" data-height="300" data-default-tab="html,result" data-slug-hash="gOjpJOo" data-user="bgzo" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
  <span>See the Pen <a href="https://codepen.io/bgzo/pen/gOjpJOo">
  Untitled</a> by bGZo (<a href="https://codepen.io/bgzo">@bgzo</a>)
  on <a href="https://codepen.io">CodePen</a>.</span>
</p>

### Hyperlink

<p class="codepen" data-height="300" data-default-tab="html,result" data-slug-hash="KKBpLpE" data-user="bgzo" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
  <span>See the Pen <a href="https://codepen.io/bgzo/pen/KKBpLpE">
  Hyperlink</a> by bGZo (<a href="https://codepen.io/bgzo">@bgzo</a>)
  on <a href="https://codepen.io">CodePen</a>.</span>
</p>

[^1]: [Emojis as favicons](https://css-tricks.com/emojis-as-favicons);
[^3]: Basically I agree with magasa's answer, via: [哪个或哪些网络用语最让人反感？ - 知乎](https://www.zhihu.com/question/20009911/answer/15802749); Wherever it was, native or foreign, cliché sucks, why not using a more concrete and clearer word to discuss? via: [Cliché - Wikipedia](https://en.wikipedia.org/wiki/Clich%c3%a9) or [Platitude - Wikipedia](https://en.wikipedia.org/wiki/Platitude). Since 20200501;
[^4]: As laike9m said, "对抗死亡是人类文明的永恒主题，而我们已经实现了阶段性胜利。现在是文章、照片、视频，也许还有以个人习惯作为输入训练的模型。再往后呢？会不会有基因信息，乃至意识的完整复制呢？依托于稳定的存储，我们能做的事情实在太多了。". via: [People Die, but Long Live GitHub - laike9m's blog](https://laike9m.com/blog/people-die-but-long-live-github,122/);
[^5]: [Heaven's Lost Property - Wikipedia](https://en.wikipedia.org/wiki/Heaven%27s_Lost_Property);
[^6]: [全平台 Favicon 在线生成器 – RealFaviconGenerator.net – 镜态极研](https://kagamiz.com/real-favicon-generator/);
[^7]: See sense via: [BoJack Learns How Hard It Is to Write - BoJack Horseman - YouTube](https://www.youtube.com/watch?v=tOxB_Dlb-rA). "I feel like we can beat this font. Ah, Copperplate, a font for the truly modern man. Ha. No. There you are, Verdana. Oh, Comic Sans, you drunk, go home. 3:10? I just spent six hours playing with fonts? Holy shit!" Generally, every decision is hard and diversity but not matters. Since 20210607;
[^8]: "一个很自然的便捷度区分出现了：备案或白名单网站只需要点击一次就可以查看；非备案或非白名单网站（如很多国际网站）需要多点击一次。仅仅这一点，恐怕就可以让很多人放弃访问这个页面（因为便捷度弹性高）。何况这个跳转页面有意无意间说到了「风险」两个字，无疑又会阻止一部分人。", via: [软墙 - 郝海龍](https://haohailong.net/2021/soft-wall )

<script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>

**Comment by bGZo** [comment]
![WindowsTerminal_1ibNbTPfwq](https://user-images.githubusercontent.com/57313137/158712463-0f350cb8-e83b-41cf-a90b-6fa5a93b0113.png)
<img src="https://user-images.githubusercontent.com/57313137/158712544-96fcd594-7628-41e8-a906-acdc672d5e22.png" width=45%><img src="https://user-images.githubusercontent.com/57313137/158712547-68a408d5-a46d-42ec-ab6b-35f1f8a3af55.png" width=45%>
